### PR TITLE
Added React and Main page

### PR DESCRIPTION
**Pull request by fabionobre**
I added ReactJS lib in Gemfile, configured ReactJS and created the Main page. 
I added MaterializeCSS to design and configure the basic design on the Main page.

### CODE REVIEW

File: app/helpers/site_helper.rb [hunk 1]
```diff
@@ -0,0 +1,2 @@
+module SiteHelper
```
Comment (marceloboth) [inline app/helpers/site_helper.rb]: You can remove this file. In my opinion, we'll never use it.

File: spec/helpers/site_helper_spec.rb [hunk 2]
```diff
@@ -0,0 +1,15 @@
+require 'rails_helper'
```
Comment (marceloboth) [inline spec/helpers/site_helper_spec.rb]: This one can also be removed .

File: spec/views/site/index.html.erb_spec.rb [hunk 3]
```diff
@@ -0,0 +1,5 @@
+require 'rails_helper'
```
Comment (marceloboth) [inline spec/views/site/index.html.erb_spec.rb]: This one can also be removed.

File: app/helpers/site_helper.rb (hunk 1 above)
Comment (fabionobre) [inline app/helpers/site_helper.rb]: Yeah, you are right. I will do this in the next commit.

File: app/controllers/api/v1/groups_controller.rb [hunk 4]
```diff
@@ -2,7 +2,7 @@ class Api::V1::GroupsController < ApplicationController
   def index
     @groups = Group.all
 
-    render json: @groups
+    render json: @groups, :callback => params[:callback]
```
Comment (marceloboth) [inline app/controllers/api/v1/groups_controller.rb]: My suggestion here is using `callback: params[:callback]` instead. It's only a sintax sugar, not a big deal 😄

**Pull request merged**